layout/theme.liquid

```diff
@@ -132,15 +132,30 @@
     {% include 'cart-popup' %}
   {%- endif -%}
 
-  {% section 'header' %}
+
+{% comment %}======= Hiding Header from cart pagewith conditional {% endcomment %}
+
+  {% unless template.suffix contains "cart" %}
```

Comment:
Most likely not working due to template file name being cart.liquid (no suffix, i.e. page.suffix.liquid).  Change Conditional.